tests/__fixtures__/cardanoSignTransaction.js

```diff
@@ -309,6 +328,7 @@ const SAMPLE_MINTS = {
 const FEE = '42';
 const TTL = '10';
 const VALIDITY_INTERVAL_START = '47';
+const SCRIPT_DATA_HASH = '3b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b7';
```

Comment:
Please use a different hash than is used for inputs.

Reply:
Fixed.